geometry_msgs/PoseStamped left_pose
geometry_msgs/PoseStamped right_pose